deployment/ccip/changeset/testhelpers/messagingtest/helpers.go

```diff
@@ -153,18 +154,29 @@ func Run(t *testing.T, tc TestCase) (out TestCaseOutput) {
 	var msg any
 	switch family {
 	case chain_selectors.FamilyEVM:
+		feeToken := common.HexToAddress("0x0")
+		if len(tc.FeeToken) > 0 {
+			feeToken = common.HexToAddress(tc.FeeToken)
+		}
+
 		msg = router.ClientEVM2AnyMessage{
 			Receiver:     common.LeftPadBytes(tc.Receiver, 32),
 			Data:         tc.MsgData,
 			TokenAmounts: nil,
-			FeeToken:     common.HexToAddress("0x0"),
+			FeeToken:     feeToken,
 			ExtraArgs:    tc.ExtraArgs,
 		}
 	case chain_selectors.FamilySolana:
+		feeToken := solana.PublicKey{}
+		if len(tc.FeeToken) > 0 {
+			feeToken = solana.MustPublicKeyFromBase58(tc.FeeToken)
```

Comment:
nit: could we use the non-panic method here? Might be easier to track any errors 
```suggestion
			feeToken, err := solana.PublicKeyFromBase58(tc.FeeToken)
			require.NoError(t, err)
```

Reply:
Good call thx! Should be fixed now